docs/developer/shipping/shipping-method.mdx

```diff
@@ -0,0 +1,424 @@
+---
+title: Shipping Methods
+description: Overview of shipping methods business logic and configuration in Saleor.
+---
+
+# Shipping Methods
+
+Shipping Methods define the available delivery options and their associated costs within a specific shipping zone 
+They define how much customers pay, what carriers you use, and under what conditions each method applies.
+
+Saleor supports two types of shipping methods:
+- **Price-based**: These methods are available when the order total falls 
+within the configured minimum and maximum price range.
+- **Weight-based**: These methods are available when the order's total weight falls 
+within the configured minimum and maximum weight range.
+
+
+:::note
+In Saleor, **Shipping Methods** and **Shipping Rates** refer to the same concept. 
+When creating them in the dashboard, they are called Shipping Rates, 
+while in the API responses they are returned as Shipping Methods.
+:::
+
+
+:::tip
+For details on using webhooks to integrate external shipping methods, 
```

Comment:
> integrate external shipping methods,

adding external shipping methods is one webhook, but we also have shippign-filter webhook. Maybe worth to be more generic here? Like: 

> For details on using shipping webhooks see the [Shipping webhooks chapter]

WDYT?